cdap-support-bundle/bin/observability.py

```diff
@@ -0,0 +1,539 @@
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/log.saver/logs?start=0 -H "{auth_token}" ',
+            "fileName": 'kubectl-exec-log-saver-logs-from-api',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/runtime/logs?start=0 -H "{auth_token}" ',
+            "fileName": 'kubectl-exec-runtime-logs-from-api',
+            "fileExt": 'log',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_logs
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/system/services/status -H "{auth_token}"',
+            "fileName": 'kubectl-exec-system-service-status',
+            "fileExt": 'txt',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_status
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/namespaces/system/apps/pipeline/services/studio/status -H "{auth_token}"',
+            "fileName": 'kubectl-exec-system-service-studio-status',
+            "fileExt": 'txt',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_status
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/namespaces/system/apps/dataprep/services/service/status -H "{auth_token}"',
+            "fileName": 'kubectl-exec-system-service-wrangler-status',
+            "fileExt": 'txt',
+            "fileWrite": True,
+            "forEachNode": False,
+            "forEachCommand": '',
+            "placeholderString": "",
+            "parentFolder": cdap_services_status
+        },
+        {
+            "command": f'kubectl exec cdap-{instance_name}-runtime-0 -n {namespace} -- curl cdap-{instance_name}-router:11015/v3/namespaces/{cdap_ns}/apps/{pipeline_name}/workflows/DataPipelineWorkflow/runs/{run_id}/logs -H "Authorization: Bearer {auth_token}"',
```

Comment:
what happens if cdap_ns is not specified?

Reply:
Command will be skipped, if cap_ns is not specified